codec-http/src/main/java/io/netty/handler/codec/http/CombinedHttpHeaders.java

```diff
@@ -213,9 +215,13 @@ public CombinedHttpHeadersImpl setObject(CharSequence name, Iterable<?> values)
             return this;
         }
 
+        private static boolean cannotBeCombined(CharSequence name) {
+            return CASE_SENSITIVE_HASHER.equals(SET_COOKIE, name);
```

Comment:
I think you could just use `SET_COOKIE.contentEqualsIgnoreCase(...)`.

Reply:
Fixed!